### ISSUE TITLE
Modern Vulkan-Loader does not load multiarch Mangohud layer

### ISSUE THREAD

**Issue by Atemu** [flightlessmango/MangoHud#1021]
**Describe the bug**

The Vulkan-Loader skips attempting to load the 64-bit layer after it found the 32-bit layer to have the wrong bitness.

```
INFO | LAYER:     Requested layer "VK_LAYER_MANGOHUD_overlay" was wrong bit-type.
```

Changing the lmplicit layer's name in the JSON to have a bitness-specific suffix fixes the issue.

This was noticed in https://github.com/NixOS/nixpkgs/issues/230978 when https://github.com/NixOS/nixpkgs/pull/228870 updated [vulkan-loader: 1.3.243 -> 1.3.249](https://github.com/NixOS/nixpkgs/pull/228870/commits/81b9269df9ec371f36423e3d9a2f76c5a3ea1d7c). @K900 has chased down the likely upstream culprit to be https://github.com/KhronosGroup/Vulkan-Loader/commit/c3eb97de22e9e2eec061be897dcf13f2c0759788.  
We plan on [patching around the issue](https://github.com/NixOS/nixpkgs/pull/231874) for the time being but wanted to notify you guys too.

**List relevant hardware/software information**
- Linux Distribution
- MangoHud version
- GPU

**To Reproduce**

**Expected behavior**

**Screenshots**

**Additional context**

**Comment by flightlessmango** [comment]
I believe this commit fixes this issue 5754ca1

**Comment by Atemu** [comment]
Thanks, we'll remove the workaround in the next release!
